app/pages/resource/_resource-page.scss

```diff
@@ -138,50 +143,32 @@ $content-padding: 10px;
     margin-right: -15px;
   }
 
-  /* Full width page */
   @media (min-width: $screen-xs-max) {
-    /* Fix for resource page */
-      /* Padding for turquise area */
-      .app-ResourceHeader__content {
-         padding: 15px 15px 15px 15px;
-      }
-      /*smaller header*/
-      .h1, h1 {
-         font-size: 32px;
-      }
-      /*basetext fix*/
-      .row {
-         margin-left: 15px;
-         margin-right: 15px;
-         margin-top: 15px;
-      }
-      .app-ResourcePage__content {
-         padding: 0px;
-      }
+    .app-ResourceHeader__content {
+      padding: 15px 15px 15px 15px;
+    }
+
+    .h1, h1 {
+      font-size: 32px;
+    }
+
+    .app-ResourcePage__content {
+      padding: 0;
+    }
   }
   @media (max-width: $screen-xs-max) {
-      /* Footer must be below map or other content */
-      #footer {
-         position: relative;
-         z-index: -15;
-      }
-      /* Fix for resource page */
-      /* Padding for turquise area */
-      .app-ResourceHeader__content {
-         padding: 15px 15px 15px 15px;
-      }
-      /* smaller header */
-      .h1, h1 {
-         font-size: 24px;
-      }
-      /* basetext fix */
-      .row {
-         margin-left: 15px;
-         margin-right: 15px;
-         margin-top: 15px;
-      }
-      .app-ResourcePage__content {
-         padding: 0px;
-      }
-   }
+    #footer {
+      position: relative;
+      z-index: -15;
+    }
+    .app-ResourceHeader__content {
+      padding: 15px 15px 15px 15px;
```

Comment:
`padding: 15px` ?

Reply:
not my style mate :)

Reply:
Its same tho, but shorter :D

Reply:
I just did some cleaning up for the existing styles. I'm not going to start fixing every issue in them :)